jax/lax/lax_control_flow.py

```diff
@@ -1096,7 +1096,7 @@ def cond_bind(*args, branches, linear):
 
 ### scan
 
-def scan(f, init, xs, length=None, reverse=False):
+def scan(f, init, xs, length=None, reverse=False, _unroll=None):
```

Comment:
Might it not be simpler to default to `1`, rather than supporting a `None` option? This would also allow you to just unconditionally assume it exists rather than using `**params` everywhere.